src/preprocessor.ts

```diff
@@ -58,7 +60,15 @@ export function process(
 
   const modified = injectSourcemapHook(outputText);
 
-  cacheFile(jestConfig, filePath, modified);
+  if (tsJestConfig.enableInternalCache === true) {
```

Comment:
So if I understand correct, enableInternalCache says we don't cache things, and is undefined/falsey per default?

Reply:
yes. Unless set in the config, this flag will be falsy (not `true`, to be precise)